go/stellar/relay.go

```diff
@@ -0,0 +1,123 @@
+package stellar
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/client/go/protocol/stellar1"
+	"github.com/keybase/client/go/stellar/remote"
+	"github.com/keybase/client/go/teams"
+	"github.com/keybase/stellarnet"
+	"github.com/stellar/go/build"
+	"github.com/stellar/go/keypair"
+	"golang.org/x/crypto/nacl/secretbox"
+)
+
+// Get the key used to encrypted the key for a relay transfer
+// A key from the implicit team betwen the logged-in user and `to`.
+// If `generation` is nil, gets the latest key.
+// TODO CORE-7718 make this private
+func KeyForRelayTransfer(ctx context.Context, g *libkb.GlobalContext, remoter remote.Remoter,
+	other *libkb.User, generation *keybase1.PerTeamKeyGeneration) (res keybase1.TeamApplicationKey, err error) {
```

Comment:
Instead of `other` as a libkb.User, what about just a user assertion?  Would that magically make SBS work?

Reply:
replaced with different non-magic